ui/components/User.js

```diff
@@ -48,72 +47,73 @@ function exportToJsonFile(jsonData, filename) {
   linkElement.remove()
 }
 
-class User extends React.Component {
-  state = {
-    user : null,
-    open : false,
-    account : ExtensionPointSchemaValidator("account")(),
-    providerType : '',
-    capabilitiesLoaded : false
-  }
+function User(props){
+  const [user, setUser] = useState(null);
+  const [open, setOpen] = useState(false);
+  const [account, setAccount] = useState([]);
+  // eslint-disable-next-line no-unused-vars
+  const [providerType, setProviderType] = useState('');
```

Comment:
@Mohith234 why we silencing this eslint warning if we're not using this state shouldn't we remove it

Reply:
We are updating the state of `providerType` using the `setProviderType` on line number **`109`**. We are not using the `providerType` variable directly so the eslint was giving me a warning.

I have silenced the warning again for the handleError function on line number **`78`**. I see no usage of this function in the previous class component too. Should I remove it? @Yashsharma1911

Reply:
Please remove both if we are not using them, instead silencing the lint warnings

Reply:
I think other things looks good to me, nice work here @Mohith234 👏

Reply:
- I will remove the `handleError` function but what about `setProviderType`? 
- It must be present because of Line number [109](https://github.com/meshery/meshery/pull/8834/files#diff-f35ed20febe118e563e725e93c25c07534f96a3bd3dcb031b05fa661932522c8R109) @Yashsharma1911

Reply:
Remove it too, cause it doesn't make sense of setting state which we're not using